src/ol/math.js

```diff
@@ -79,6 +79,69 @@ ol.math.squaredDistance = function(x1, y1, x2, y2) {
 
 
 /**
+ * Solves system of linear equations using Gaussian elimination method.
+ *
+ * @param {Array.<Array.<number>>} A Augmented matrix (n x n + 1 column)
+ *                                   in row-major order.
+ * @return {Array.<number>} The resulting vector.
+ */
+ol.math.solveLinearSystem = function(A) {
```

Comment:
In know in maths matrices are written with a capital letter, but usually we start variable names in lower-case (also for matrices, for exampe [here](https://github.com/openlayers/ol3/blob/master/src/ol/vec/mat4.js#L15))